UI/window-basic-main.cpp

```diff
@@ -2109,6 +2098,8 @@ void OBSBasic::OnFirstLoad()
 
 	if (showLogViewerOnStartup)
 		on_actionViewCurrentLog_triggered();
+
+	ResetUI();
```

Comment:
Is this general bug fix? Or this is part of the new feature?
(in case of bug fix, can you link the issue #)

Reply:
This is part of the new PR.

Reply:
It has been removed now.